_dashboards/branding.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: default
+title: Custom branding
+nav_order: 200
+---
+
+# Dashboards custom branding
+Introduced 1.2
+{: .label .label-purple }
+
+By default, OpenSearch Dashboards uses the OpenSearch logo, but if you want to use custom branding elements such as the favicon or main Dashboards logo, you can do so by editing `opensearch_dashboards.yml` or by including a custom `opensearch_dashboards.yml` file when you start your OpenSearch cluster.
+
+For example, if you're using Docker to start your OpenSearch cluster, include the following lines in the `opensearch-dashboards` section of your `docker-compose.yml` file:
+
+```
+volumes:
+  - ./opensearch_dashboards.yml:/usr/share/opensearch-dashboards/config/opensearch_dashboards.yml
+```
+
+Restart your OpenSearch cluster, open OpenSearch Dashboards in your browser again, and OpenSearch Dashboards now uses your custom elements.
```

Comment:
NIT: `Restart your OpenSearch cluster`, if you mean Docker cluster I agree, if you been like OpenSearch cluster then you don't have to restart OpenSearch. Just restart OpenSearch Dashboards.